### PR TITLE
Add Alert component

### PR DESCRIPTION
**Pull request by Michaeltym**
## Change
1. Add Alert component
2. Fix failed snapshot tests (Checkbox and Radio)
## Usage
```javascript
<Alert palette="danger">Alert Message</Alert>
```
#### palette (string, optional, default is `default`)
one of default, primary, secondary, success, danger and warning
## Checklist
- [x] Documentation
- [x] Unit tests

### CODE REVIEW

File: src/themes/themes.mdx
Comment (jxom) [inline src/themes/themes.mdx:164]: Can we also add the `AlertThemeConfig` as a type in `./types.js`? :)

File: src/Alert/Alert.js
Comment (jxom) [inline src/Alert/Alert.js]: Regarding accessibility, it would be great if we could add `role="alert"` as an attribute to `<_Alert>`. See here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role

**Review by jxom — CHANGES_REQUESTED** [review]
This looks like a good start! I just have a couple of comments.

I'll merge this once the comments are resolved, but in the future it would be nice to have 'light' variants of alert that look kind of similar to: https://bulma.io/documentation/components/message/#message-body-only and https://evergreen.segment.com/components/alert. But this is a good start for the 'darker' variants of alert! Titles and icons would also be nice as well. I'll raise this as an issue once this is merged.

**Comment by Michaeltym** [comment]
Thanks @jxom for reviewing the PR. Will add the `light` version and icon/title later 👍

**Pull request merged**
